docs/get-started/getting-started-gcp.md

```diff
@@ -86,8 +86,8 @@ If you'd like to further accelerate init time to 4-5 minutes, create a custom Da
 To use notebooks with a Dataproc cluster, click on the cluster name under the Dataproc cluster tab
 and navigate to the "Web Interfaces" tab.  Under "Web Interfaces", click on the JupyterLab or
 Jupyter link to start to use sample [Mortgage ETL on GPU Jupyter
-Notebook](../demo/GCP/Mortgage-ETL-GPU.ipynb) to process full 17 years [Mortgage
-data](https://rapidsai.github.io/demos/datasets/mortgage-data).
+Notebook](../demo/GCP/Mortgage-ETL-GPU.ipynb) to process full 17 years 
```

Comment:
[Same comment](https://github.com/NVIDIA/spark-rapids/pull/4743#discussion_r803743695) I posted on the previous PR.  This link is split across lines, which the other PR said it was going to address.

Reply:
thanks, updated